pkg/rpc/server.go

```diff
@@ -225,6 +227,38 @@ Methods:
 		} else {
 			results = "Invalid public account address"
 		}
+	case "getrawtransaction":
+		var err error
+
+		param0, errParam := reqParams.ValueWithType(0, "string")
+		if errParam != nil {
+			err = errors.Wrap(errParam, "expected param at index 0 to be a valid string transactionID")
+			resultsErr = NewInvalidParamsError(err.Error(), err)
```

Comment:
you already get error in `errParam`.. don't wrap it, just pass to `resultErr`

Reply:
okay. Will do.

Reply:
That’s behavior of C# code